akka-http-core/src/main/scala/akka/http/impl/model/parser/SimpleHeaders.scala

```diff
@@ -6,7 +6,7 @@ package akka.http.impl.model.parser
 
 import akka.annotation.InternalApi
 import akka.parboiled2.Parser
-import akka.http.scaladsl.model.{ RemoteAddress, Uri }
+import akka.http.scaladsl.model.{ RemoteAddress, Uri, DateTime }
```

Comment:
It doesn't look like this import is used after all.

Reply:
I'll check and remove, in case